sympy/solvers/__init__.py

```diff
@@ -32,3 +32,7 @@
 from .decompogen import decompogen
 
 from .solveset import solveset, linsolve, linear_eq_to_matrix
+
+from ..core.singleton import S
+Complexes = S.Complexes
```

Comment:
I think this should go along with `Reals` in `sets/__init__.py`.
Is there any particular you are exposing `Complexes` here ?